pkg/imgpkg/bundle/bundle.go

```diff
@@ -117,9 +121,16 @@ func (o *Bundle) pull(baseOutputPath string, ui goui.UI, pullNestedBundles bool,
 		imagesLockUI = goui.NewNoopUI()
 	}
 
-	err = NewImagesLock(imagesLock, o.imgRetriever, o.Repo()).WriteToPath(filepath.Join(baseOutputPath, bundlePath), imagesLockUI)
-	if err != nil {
-		return fmt.Errorf("Rewriting image lock file: %s", err)
+	imagesLockUI.BeginLinef("\nLocating image lock file images...\n")
```

Comment:
since we added printing to here, do we need this method? https://github.com/vmware-tanzu/carvel-imgpkg/blob/400d8e97f430c1e800efaf21a018a636f22dbc99/pkg/imgpkg/bundle/images_lock.go#L58

Reply:
Nice spot. deleted this unused function.